test/parallel/test-zlib-invalid-arg-value-brotli-compress.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+require('../common');
+
+// This test ensures that the BrotliCompress function throws
+// ERR_INVALID_ARG_TYPE when the parameter opts is neither
+// a number nor a boolean.
```

Comment:
```suggestion
// ERR_INVALID_ARG_TYPE when the values of the `params` key-value object are
// neither numbers nor strings.
```

Reply:
@lpinca this says that the type of `value` should neither be a `number` nor a `boolean` for `Brotli` to throw the error:
https://github.com/nodejs/node/blob/ab660a541605a0a823f0c9471ff667a6620be2a5/lib/zlib.js#L806
So did you mean `boolean` instead of `string`?

Reply:
Yes, sorry.

Reply:
No problem, updated :smiley: